### PR TITLE
SUS-1276 | sunset WikiaHomePage extension

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-1276

### CODE REVIEW

File: extensions/wikia/Search/WikiaSearchAjaxController.class.php [hunk 1]
```diff
@@ -27,9 +27,7 @@ public function getNextResults() {
 		$crossWikia = $this->request->getBool( 'crossWikia' );
 		$hub = false;
 
-		$isCorporateWiki = !empty( $this->wg->EnableWikiaHomePageExt );
-
-		$isInterWiki = $crossWikia ? true : $isCorporateWiki;
+		$isInterWiki = $crossWikia ? true : false;
```
Comment (WikiaTech) [inline extensions/wikia/Search/WikiaSearchAjaxController.class.php]: MINOR Remove the literal "true" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [BodyController.class.php#L189](https://github.com/Wikia/app/blob/62d63f219afe9edc50ebf1c52a107798809fb313/skins/oasis/modules/BodyController.class.php#L189): This branch's code block is the same as the block for the branch on line 173. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [BodyController.class.php#L189](https://github.com/Wikia/app/blob/766e234265ad981ad579e49cf8d4561b70257e8a/skins/oasis/modules/BodyController.class.php#L189): This branch's code block is the same as the block for the branch on line 173. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [BodyController.class.php#L189](https://github.com/Wikia/app/blob/e3946c137bc76fcbaef2ecb4dbb766ee1ef6ce60/skins/oasis/modules/BodyController.class.php#L189): This branch's code block is the same as the block for the branch on line 173. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [BodyController.class.php#L189](https://github.com/Wikia/app/blob/cad0ed2febd3f5455ee3dd6e22f6f4d100bc2dde/skins/oasis/modules/BodyController.class.php#L189): This branch's code block is the same as the block for the branch on line 173. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by mszabo-wikia — APPROVED** [review]

File: extensions/wikia/Search/WikiaSearchAjaxController.class.php [hunk 2]
```diff
@@ -27,6 +27,7 @@ public function getNextResults() {
 		$crossWikia = $this->request->getBool( 'crossWikia' );
 		$hub = false;
 
+		// TODO: refactor to avoid using wgEnableWikiaHomePageExt
```
Comment (WikiaTech) [inline extensions/wikia/Search/WikiaSearchAjaxController.class.php:30]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: extensions/wikia/Search/WikiaSearchController.class.php [hunk 3]
```diff
@@ -881,7 +881,8 @@ protected function templateExists( $name ) {
 	 * @see SearchControllerTest::testIsCorporateWiki
 	 */
 	protected function isCorporateWiki() {
-		return WikiaPageType::isCorporatePage();
+		// TODO: refactor to avoid using wgEnableWikiaHomePageExt
```
Comment (WikiaTech) [inline extensions/wikia/Search/WikiaSearchController.class.php:884]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**